sycl/include/sycl/ext/oneapi/raw_kernel_arg.hpp

```diff
@@ -0,0 +1,34 @@
+//==--- raw_kernel_arg.hpp --- SYCL extension for raw kernel args ----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <stddef.h>
```

Comment:
Should we use `#include <cstddef>` instead? C headers pollute the global namespace, so I try to avoid them as much as possible.

Reply:
Though I agree with the problem of global namespace pollution, this will not be the first case of `<stddef.h>`. It may be because the SYCL specification mentions `size_t` without the `std::` qualification. If it is something we would want to change though, we should do it at a project-wide level.